be/src/vec/exec/scan/vscan_node.cpp

```diff
@@ -402,7 +402,7 @@ Status VScanNode::_normalize_conjuncts() {
             RETURN_IF_ERROR(_normalize_predicate(conjunct->root(), conjunct.get(), new_root));
             if (new_root) {
                 conjunct->set_root(new_root);
-                if (_should_push_down_common_expr()) {
+                if (_should_push_down_common_expr() && !conjunct->root()->is_java_udf()) {
```

Comment:
if the new root is   javaudf expr or javaudf expr, then the new root is a and compound expr. It could not be pushdown to scanner ?